Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/AgendaDayFragment.kt

```diff
@@ -170,11 +169,11 @@ class AgendaDayFragment : Fragment(), FlexibleAdapter.OnItemClickListener {
             val agendaDetailFragment = AgendaDetailFragment()
             agendaDetailFragment.arguments = arguments
 
-            val fragmentManager = activity!!.fragmentManager
-            fragmentManager.beginTransaction()
-                    .add(R.id.fragment_container, agendaDetailFragment)
-                    .addToBackStack(null)
-                    .commit()
+            val fragmentManager = activity?.fragmentManager
+            fragmentManager?.beginTransaction()
+                    ?.add(R.id.fragment_container, agendaDetailFragment)
+                    ?.addToBackStack(null)
```

Comment:
Why call addToBackStack? Been wondering why null is passed in...

Reply:
So that when u hit back, you are taken to list.